Power Up Proto/src/org/usfirst/frc/team5493/robot/subsystems/OneClimbyBoi.java

```diff
@@ -5,66 +5,106 @@
 
 import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
 
-import edu.wpi.first.wpilibj.CounterBase.EncodingType;
-import edu.wpi.first.wpilibj.Encoder;
+import edu.wpi.first.wpilibj.Counter;
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.Preferences;
 import edu.wpi.first.wpilibj.command.Subsystem;
 
 /**
  *
  */
 public class OneClimbyBoi extends Subsystem {
 
-	public WPI_TalonSRX climbMotor;
-	public Object climbDown;
-	
-	public static final double kDistancePerRevolution = 18.84;
-	public static final double kPulsesPerRevolution = 1024;
-	public static final double kDistancePerPulse = kDistancePerRevolution / kPulsesPerRevolution;
+	//TODO: Hold can be removed when PID is integrated.
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)